src/testshade/CMakeLists.txt

```diff
@@ -2,6 +2,8 @@
 # SPDX-License-Identifier: BSD-3-Clause
 # https://github.com/AcademySoftwareFoundation/OpenShadingLanguage
 
+set (LLVM_COMPILE_FLAGS ${LLVM_COMPILE_FLAGS} "-std=c++17")
```

Comment:
This one doesn't feel right. Shouldn't it use whatever standard is used by the rest of the codebase, instead of hard-coding it to 17?

Reply:
That sounds like a good request for an improvement to this change, and I'm CC'ing @krohmerNV for his thoughts.

Reply:
As an initial proposal for this improvement, perhaps it could be written as follows?

```
set (LLVM_COMPILE_FLAGS ${LLVM_COMPILE_FLAGS} -std=c++${CMAKE_CXX_STANDARD})
```

Reply:
I've made the change described above, and have verified that the OSL build for Windows works as before.